### ISSUE TITLE
Manual Import of Unmapped Files - Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by mels0n** [Readarr/Readarr#1425]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When selecting Manual Import for a file listed in Unmapped Files an error is returned. 

"Object reference not set to an instance of an object."

### Expected Behavior

Import screen to verify/select book that is being imported. 

### Steps To Reproduce

Manual copy books to monitored location folder. 
Wait for them to show in Unmapped Files or run scan disk task.
From Unmapped Files tab attempt to manually import a file. 

### Environment

```markdown
- OS: Docker
- Readarr: 0.1.0.1142
- Docker Install: 0.1.0.1142-ls56 by linuxserver.io
- Using Reverse Proxy: No
- Browser: Chrome
```

### What branch are you running?

Nightly

### Trace Logs?

Standard Logs: 
```

2021-12-21 12:53:32.8|Info|DiskScanService|Scanning /mnt/media/books/
2021-12-21 12:53:33.4|Info|ImportDecisionMaker|Reading file 1/13
2021-12-21 12:53:33.4|Info|ImportDecisionMaker|Reading file 2/13
2021-12-21 12:53:33.4|Info|ImportDecisionMaker|Reading file 3/13
2021-12-21 12:53:33.4|Info|ImportDecisionMaker|Reading file 4/13
2021-12-21 12:53:33.4|Info|ImportDecisionMaker|Reading file 5/13
2021-12-21 12:53:33.5|Info|ImportDecisionMaker|Reading file 6/13
2021-12-21 12:53:33.5|Info|ImportDecisionMaker|Reading file 7/13
2021-12-21 12:53:33.5|Info|ImportDecisionMaker|Reading file 8/13
2021-12-21 12:53:33.5|Info|ImportDecisionMaker|Reading file 9/13
2021-12-21 12:53:33.5|Info|ImportDecisionMaker|Reading file 10/13
2021-12-21 12:53:33.5|Info|ImportDecisionMaker|Reading file 11/13
2021-12-21 12:53:33.5|Info|ImportDecisionMaker|Reading file 12/13
2021-12-21 12:53:33.5|Info|ImportDecisionMaker|Reading file 13/13
2021-12-21 12:53:33.6|Info|TrackGroupingService|Grouping 13 tracks
2021-12-21 12:53:33.6|Info|IdentificationService|Identifying book 1/1
2021-12-21 12:53:33.6|Info|ImportApprovedBooks|Importing 0 files
2021-12-21 12:57:38.0|Info|BackupService|Starting Backup
2021-12-21 12:59:29.2|Info|ImportDecisionMaker|Reading file 1/13
2021-12-21 12:59:29.3|Info|ImportDecisionMaker|Reading file 2/13
2021-12-21 12:59:29.3|Info|ImportDecisionMaker|Reading file 3/13
2021-12-21 12:59:29.3|Info|ImportDecisionMaker|Reading file 4/13
2021-12-21 12:59:29.3|Info|ImportDecisionMaker|Reading file 5/13
2021-12-21 12:59:29.3|Info|ImportDecisionMaker|Reading file 6/13
2021-12-21 12:59:29.3|Info|ImportDecisionMaker|Reading file 7/13
2021-12-21 12:59:29.3|Info|ImportDecisionMaker|Reading file 8/13
2021-12-21 12:59:29.3|Info|ImportDecisionMaker|Reading file 9/13
2021-12-21 12:59:29.3|Info|ImportDecisionMaker|Reading file 10/13
2021-12-21 12:59:29.3|Info|ImportDecisionMaker|Reading file 11/13
2021-12-21 12:59:29.3|Info|ImportDecisionMaker|Reading file 12/13
2021-12-21 12:59:29.3|Info|ImportDecisionMaker|Reading file 13/13
2021-12-21 12:59:29.4|Info|TrackGroupingService|Grouping 13 tracks
2021-12-21 12:59:29.4|Info|IdentificationService|Identifying book 1/1
2021-12-21 12:59:29.4|Error|Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware|An unhandled exception has occurred while executing the request.

[v0.1.0.1142] System.NullReferenceException: Object reference not set to an instance of an object.
   at Readarr.Api.V1.ManualImport.ManualImportResourceMapper.ToResource(ManualImportItem model) in D:\a\1\s\src\Readarr.Api.V1\ManualImport\ManualImportResource.cs:line 35
   at System.Linq.Enumerable.SelectListIterator`2.ToList()
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Readarr.Api.V1.ManualImport.ManualImportResourceMapper.ToResource(IEnumerable`1 models) in D:\a\1\s\src\Readarr.Api.V1\ManualImport\ManualImportResource.cs:line 63
   at Readarr.Api.V1.ManualImport.ManualImportController.GetMediaFiles(String folder, String downloadId, Nullable`1 authorId, Boolean filterExistingFiles, Boolean replaceExistingFiles) in D:\a\1\s\src\Readarr.Api.V1\ManualImport\ManualImportController.cs:line 53
   at lambda_method198(Closure , Object , Object[] )
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Readarr.Http.Middleware.BufferingMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Readarr.Http\Middleware\BufferingMiddleware.cs:line 28
   at Readarr.Http.Middleware.IfModifiedMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Readarr.Http\Middleware\IfModifiedMiddleware.cs:line 41
   at Readarr.Http.Middleware.CacheHeaderMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Readarr.Http\Middleware\CacheHeaderMiddleware.cs:line 33
   at Readarr.Http.Middleware.UrlBaseMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Readarr.Http\Middleware\UrlBaseMiddleware.cs:line 27
   at Readarr.Http.Middleware.VersionMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Readarr.Http\Middleware\VersionMiddleware.cs:line 28
   at Microsoft.AspNetCore.ResponseCompression.ResponseCompressionMiddleware.InvokeCore(HttpContext context)
   at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationResult authorizeResult)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)


2021-12-21 12:59:29.4|Fatal|ReadarrErrorPipeline|Request Failed. GET /api/v1/manualimport

[v0.1.0.1142] System.NullReferenceException: Object reference not set to an instance of an object.
   at Readarr.Api.V1.ManualImport.ManualImportResourceMapper.ToResource(ManualImportItem model) in D:\a\1\s\src\Readarr.Api.V1\ManualImport\ManualImportResource.cs:line 35
   at System.Linq.Enumerable.SelectListIterator`2.ToList()
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Readarr.Api.V1.ManualImport.ManualImportResourceMapper.ToResource(IEnumerable`1 models) in D:\a\1\s\src\Readarr.Api.V1\ManualImport\ManualImportResource.cs:line 63
   at Readarr.Api.V1.ManualImport.ManualImportController.GetMediaFiles(String folder, String downloadId, Nullable`1 authorId, Boolean filterExistingFiles, Boolean replaceExistingFiles) in D:\a\1\s\src\Readarr.Api.V1\ManualImport\ManualImportController.cs:line 53
   at lambda_method198(Closure , Object , Object[] )
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Readarr.Http.Middleware.BufferingMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Readarr.Http\Middleware\BufferingMiddleware.cs:line 28
   at Readarr.Http.Middleware.IfModifiedMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Readarr.Http\Middleware\IfModifiedMiddleware.cs:line 41
   at Readarr.Http.Middleware.CacheHeaderMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Readarr.Http\Middleware\CacheHeaderMiddleware.cs:line 33
   at Readarr.Http.Middleware.UrlBaseMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Readarr.Http\Middleware\UrlBaseMiddleware.cs:line 27
   at Readarr.Http.Middleware.VersionMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Readarr.Http\Middleware\VersionMiddleware.cs:line 28
   at Microsoft.AspNetCore.ResponseCompression.ResponseCompressionMiddleware.InvokeCore(HttpContext context)
   at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationResult authorizeResult)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
```
[trace.txt](https://github.com/Readarr/Readarr/files/7757477/trace.txt)

[AB#2241](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/2241)

**Comment by SoCraDi** [comment]
I had the same issue with an audiobook manual import. I looked at the debug log and noticed that incorrect "candidates" (releases?) were selected to match against (random book summaries were being selected - see redacted log below). Then I realized that the ID3 tag of the mp3 file had artist = author name and NOT artist = author name (narrator name). I updated the tag and it was able to find the right releases and import successfully.

Debug log:
2021-12-22 22:10:19.3|Debug|IdentificationService|Matching 1 track files against candidates
2021-12-22 22:10:19.3|Debug|IdentificationService|Trying Release [redacted no.][Summary And Analysis Of: Book Name]
2021-12-22 22:10:19.3|Debug|IdentificationService|Release [redacted no.][Summary And Analysis Of: Book Name] has distance 0.8316326530612246 vs best distance 1 [0ms]

2021-12-22 22:37:56.8|Debug|IdentificationService|Trying Release [redacted no2.][Book Name: The Complete Summary And Analysis]
2021-12-22 22:37:56.8|Debug|IdentificationService|Release [redacted no2.][Book Name: The Complete Summary And Analysis] has distance 0.6964285714285715 vs best distance 0.8316326530612246 [0ms]
